app/templating/summary/answer.py

```diff
@@ -1,7 +1,9 @@
 class Answer:
 
-    def __init__(self, answer_schema, answer):
+    def __init__(self, answer_schema, answer, other_value=None):
         self.id = answer_schema['id']
         self.label = answer_schema['label'] if 'label' in answer_schema else None
         self.value = answer
         self.type = answer_schema['type'].lower()
+        self.parent_answer_id = answer_schema['parent_answer_id'] if 'parent_answer_id' in answer_schema else None
+        self.other_value = other_value
```

Comment:
Same as above. Make sense to call it `child_answer_value` so it's clearer from the schema what's being talked about.